packages/forms/src/components/FilterMenu/index.tsx

```diff
@@ -141,7 +144,7 @@ export default function FilterMenu({
             <T k="lunar.common.apply" phrase="Apply" context="Apply filters button label" />
           </Link>
 
-          <Link muted type="reset" onClick={handleClear}>
+          <Link muted={mutedReset} type="reset" onClick={handleClear}>
```

Comment:
is this related to a design change? it's been a standard pattern to have the clear/reset be muted

Reply:
it's a want for profiles2 / sending you jira in PM